README.md

```diff
@@ -85,6 +85,8 @@ export class AppModule {}
 | Input               | Type    | Default           | Description                                       |
 | ------------------- | ------- | ----------------- | ------------------------------------------------- |
 | animate             | boolean | `true`            | `false` to render with no animation               |
+| baseUrl             | string  | ``                | Required if you're using `<base url="/" />` in your `<head/>`. Defaults to an empty string. This prop is commom used as: `<ContentLoader baseUrl={window.location.pathname} />` which will fill the svg attribute with the relative path. Related [#93](https://github.com/danilowoz/react-content-loader/issues/93).              
```

Comment:
Please change it to Angular component syntax.

Reply:
Done.